client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -299,6 +302,27 @@ export class ProductPurchaseFeaturesList extends Component {
 		);
 	}
 
+	getJetpackSecurityFeatures() {
+		const { isAutomatedTransfer, isPlaceholder, selectedSite } = this.props;
+		return (
+			<Fragment>
+				<SiteActivity />
+				<MonetizeSite selectedSite={ selectedSite } />
+				<MobileApps onClick={ this.handleMobileAppsClick } />
+				<JetpackPublicize selectedSite={ selectedSite } />
+				<SellOnlinePaypal isJetpack />
+				<GoogleAnalyticsStats selectedSite={ selectedSite } />
+				<FindNewTheme selectedSite={ selectedSite } />
+				<HappinessSupportCard
+					isJetpack={ !! selectedSite.jetpack && ! isAutomatedTransfer }
+					isPlaceholder={ isPlaceholder }
+					showLiveChatButton
+					liveChatButtonEventName={ 'calypso_livechat_my_plan_jetpack_professsional' }
```

Comment:
I think it would be better if we track this event as `'calypso_livechat_my_plan_jetpack_security'`.